js/securesubmit/checkout-form.js

```diff
@@ -711,6 +856,15 @@ document.observe('dom:loaded', function () {
                         }
                     }
                 });
+
+                if (window.amastyCompleteCheckoutOriginal && document.querySelector('#amscheckout-onepage')) {
+
+                    document.querySelector('#ss-banner').style.backgroundSize = '325px 40px';
+                    document.querySelector('#cc-number').className = 'amasty_one_page_checkout';
+                    document.querySelector('#expiration-date').className = 'amasty_one_page_checkout';
+                    document.querySelector('#ccv').className = 'amasty_one_page_checkout';
```

Comment:
fair enough on the verification but the `document.querySelector('#node-id')` is legit syntax. do you have a reason to prefer one over the other?